www/secmail/views/forms/icla.js.rb

```diff
@@ -89,6 +89,7 @@ def componentDidMount()
     end
 
     @realname = name
+    @pubname = name
```

Comment:
The current secmail-icla has the additional (clear form) button which solves cases 3 and 4.

When I process a burst/stapled form I click (o) icla and then click (clear form). Away we go.